app/components/TransactionModal.tsx

```diff
@@ -34,7 +37,14 @@ export const TransactionModal = ({
   return (
     <Dialog open={isModalOpen(ModalId.WAITING_TRANSACTION)} onOpenChange={close}>
       <DialogContent>
-        <DialogHeader />
+        <DialogHeader>
+          <VisuallyHidden asChild>
+            <>
+              <DialogTitle>Transaction Modal</DialogTitle>
+              <DialogDescription />
```

Comment:
do we need the `DialogDescription`?

Reply:
Yes, both Title and Description are required by Content:
* The Title missing triggers an error severity console message
* The Description missing triggers a warning severity console message

Placing both components inside the Content DOM tree removes the error and the warning.
`VisuallyHidden` can only have 1 children, that's why I used a Fragment to wrap both the Title and the Description